page_objects/Base_Page.py

```diff
@@ -6,13 +6,10 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.common.action_chains import ActionChains
-import unittest,time,logging,os,inspect,pytest
+import unittest,time,logging,os,inspect
```

Comment:
Can you check if we need the `unittest` module and remove it if we are not using it?

Reply:
class Base_Page(Borg,unittest.TestCase):
    "Page class that all page models can inherit from"

Unit test is being used inside the page .

Reply:
We don't need `unittest.TestCase` Drishya. We can remove it, we are using `pytest`

Reply:
pytest I removed as it was not using in this module anywhere . Do you mean I should include pytest and remove unittest?